Makefile

```diff
@@ -183,7 +183,7 @@ test-drenv: ## Run drenv tests.
 test-ramenctl: ## Run ramenctl tests.
 	$(MAKE) -C ramenctl
 
-e2e-rdr: generate manifests docker-build ## Run rdr-e2e tests.
+e2e-rdr: generate manifests ## Run rdr-e2e tests.
```

Comment:
I am wondering if we need to retain this, cases where we run/re-run e2e in local setups we need the docker-build here. But, yes this does the build twice from the action perspective.

At present e2e script has no per-requisites (for example env etc.) so we should be ok for now.